From c503a88127502f57e15c1d85a934586bf3d7ccb7 Mon Sep 17 00:00:00 2001
From: Tao Huang <huangtao@rock-chips.com>
Date: Mon, 10 Sep 2018 16:42:56 +0800
Subject: [PATCH] soc: rockchip: opp_select: do not use _of_get_opp_desc_node

Which is come from Linaro. Use of_parse_phandle directly.

Change-Id: I7ca9de46e4ec88011165862936c00cb1b354907f
Signed-off-by: Tao Huang <huangtao@rock-chips.com>
---
 drivers/soc/rockchip/rockchip_opp_select.c | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/drivers/soc/rockchip/rockchip_opp_select.c b/drivers/soc/rockchip/rockchip_opp_select.c
index 5480a1d521a5..07818b6bd758 100644
--- a/drivers/soc/rockchip/rockchip_opp_select.c
+++ b/drivers/soc/rockchip/rockchip_opp_select.c
@@ -1127,7 +1127,7 @@ int rockchip_init_opp_table(struct device *dev,
 	int ret = 0;
 
 	/* Get OPP descriptor node */
-	np = _of_get_opp_desc_node(dev);
+	np = of_parse_phandle(dev->of_node, "operating-points-v2", 0);
 	if (!np) {
 		dev_dbg(dev, "Failed to find operating-points-v2\n");
 		return -ENOENT;
-- 
2.35.3

